ssn/chapters/Common.html

```diff
@@ -1111,7 +1114,8 @@ <h5><a href="#SOSAExecutionCollection">sosa:ExecutionCollection</a></h5>
                 Collection (set) of one or more <a href="#SOSAExecution">Executions</a>
                 or
                 <a href="#SOSAExecutionCollection">ExecutionCollections</a>, i.e., collections may be nested.
-                <br><br>
+            </p>
+            <p>
                 The following <a href="#SOSAExecution">Execution</a> properties may appear in an
                 <a href="#SOSAExecutionCollection">ExecutionCollection</a>:
                 <a href="#SOSAhasFeatureOfInterest">hasFeatureOfInterest</a> ,
```

Comment:
Similar tweaks needed on lines below
```suggestion
                <a href="#SOSAhasFeatureOfInterest">hasFeatureOfInterest</a>,
```